standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/HiveMetaStoreClient.java

```diff
@@ -605,27 +608,44 @@ private THttpClient createHttpClient(URI store, boolean useSSL) throws MetaExcep
     String path = MetaStoreUtils.getHttpPath(MetastoreConf.getVar(conf, ConfVars.THRIFT_HTTP_PATH));
     String httpUrl = (useSSL ? "https://" : "http://") + store.getHost() + ":" + store.getPort() + path;
 
-    String user = MetastoreConf.getVar(conf, ConfVars.METASTORE_CLIENT_PLAIN_USERNAME);
-    if (user == null || user.equals("")) {
-      try {
-        LOG.debug("No username passed in config " + ConfVars.METASTORE_CLIENT_PLAIN_USERNAME.getHiveName() +
-            ". Trying to get the current user from UGI" );
-        user = UserGroupInformation.getCurrentUser().getShortUserName();
-      } catch (IOException e) {
-        throw new MetaException("Failed to get client username from UGI");
+    HttpClientBuilder httpClientBuilder = HttpClientBuilder.create();
+    String authType = MetastoreConf.getAsString(conf, ConfVars.METASTORE_CLIENT_AUTH_MODE).toLowerCase(
```

Comment:
nit: the var `METASTORE_CLIENT_AUTH_MODE` is strictly allowed to be some modes, maybe we do not need to localize here?

Reply:
@dengzhhu653 : Sorry I didn't understand your question. Can you please elaborate? 

@hsnusonic : For this config, getAsString and getVar work the same way. But its better to use `getVar` here since it enforces that defaultVal of this config should be of String type.

Reply:
sorry, I mean the the `METASTORE_CLIENT_AUTH_MODE` has been restricted to `JWT`,..., there is no need to make the value be lower-cased by Local.ROOT,  `equalsIgnoreCase` may be better for this case? Just my opinion, thanks.

Reply:
@dengzhhu653 : Makes sense. I will make the change.